torchreid/integration/sc/utils.py

```diff
@@ -294,7 +295,18 @@ def __init__(self, update_progress_callback: UpdateProgressCallback, **kwargs):
 
     def on_train_batch_end(self, batch, logs=None):
         super().on_train_batch_end(batch, logs)
-        self.update_progress_callback(self.get_progress())
+        self.update_progress_callback(self.get_progress(), logs)
+
+    def on_epoch_end(self, epoch, logs=None):
+        self.past_epoch_duration.append(time.time() - self.start_epoch_time)
+        self.__calculate_average_epoch()
+        self.update_progress_callback(self.get_progress(), logs)
```

Comment:
`update_progress_callback` should implement the following signature: `def __call__(self, progress: float, score: Optional[float] = None):` (see UpdateProgressCallback in ote_sdk). 
`logs` argument is not expected at the moment, so these changes are invalid from the sc/ote sdk perspective. In the SC environment such call could imply training container failure. I'd propose you modify sdk itself in the impt repo, before making these changes here.

Reply:
Please disregard the last comment. It's better to rename the `logs` argument to avoid confusion.

Reply:
The `on_epoch_end` is overriding `ote_sdk.usecases.reporting.callback.Callback.on_epoch_end(self, epoch, logs=None)`. Even though renaming `logs` to `score` is clearer, pylint complains about this as (W0237: arguments-renamed).